plugins/catalog/src/components/CatalogPage/DefaultCatalogPage.tsx

```diff
@@ -115,21 +109,19 @@ export function DefaultCatalogPage(props: DefaultCatalogPageProps) {
     emptyContent,
     pagination,
     ownerPickerMode,
+    filters,
   } = props;
 
   return (
     <BaseCatalogPage
       filters={
-        <>
-          <EntityKindPicker initialFilter={initialKind} />
-          <EntityTypePicker />
-          <UserListPicker initialFilter={initiallySelectedFilter} />
-          <EntityOwnerPicker mode={ownerPickerMode} />
-          <EntityLifecyclePicker />
-          <EntityTagPicker />
-          <EntityProcessingStatusPicker />
-          <EntityNamespacePicker />
-        </>
+        filters ?? (
+          <DefaultFilters
```

Comment:
There's a mild risk of confusion that if you pass in custom filters, the `initialKind`, `initiallySelectedFilter` and `ownerPickerMode` props will have no effect. But it's probably ok